src/core/config.ts

```diff
@@ -7,13 +7,15 @@ let config: Config = {
     distance: 20, // 20 pixels
     interval: 25, // 25 milliseconds
     delay: 1000, // 1 second
-    expire: 7, // 7 days
+    expire: 365, // 1 year
     ping: 60 * 1000, // 1 minute
     timeout: 5 * 60 * 1000, // 5 minutes
+    session: 10 * 60 * 1000, // 5 minutes
```

Comment:
Minor suggestion: Put some constant like Time.SecondMs, Time.MinuteMs, Time.HourMs to make these easier to read.

Reply:
Sounds good, will do and then resolve.